### DISCUSSION TITLE
SAF What's next?

### DISCUSSION THREAD

**Discussion by haarer** [GfSE/SAF-Specification#41] [Q&A]
Was wollen wir nach dem Release bewegen ?
Ideen für Themen:
-workflow beschreiben (separat, integriert..)(typical, scenario based..)
- vervollständigen (fehlende VP, proposal, )
- information items auf SAF abbilden
- Beispielmodelle
- ..

Welche Themen würdet ihr gern voran bringen?

**Comment by hugoormo** [comment]
Wir haben kein Viewpoint für die Visualisierung und Verwaltung der SAF-Stereotypen sowie für Erweiterungen an der Seite der Endanwender.
Ich schlage vor dafür ein Viewpoint zu bauen. Das Thema haben wir heute im Termin mit Jan, Eric und Sascha angesprochen.

**Reply by haarer** [reply]
Meine ersten Fragen wäre für wen dieser Viewpoint wäre und welchen Informationsbedarf er befriedigt. (die übliche Stakeholder /Concern Frage)
Solche Viewpoints - wenn man denn welche beschreiben möchte -  sollte man deutlich trennen von denen die sich mit den Inhalten von Systemmodellen befassen. 
Das verwirrt sonst die Nutzer.

Wir haben unter folgendem Link schon eine Beschreibung möglicher Zielgruppen für Dokumentation.  Da müsste man mal schauen, ob die Zielgruppe für die Ihr Viewpoints entwickeln wollt dazu passt.

**Reply by hugoormo** [reply]
Die Idee ist, dass der Nutzer das Profil erweitern will und dafür regelt die Viewpoint, wie solch eine View modelliert werden sollte. Die View soll eine Topologie des Profils darstellen, damit Erweiterungen konsistent gemacht werden können. Die View, ohne vom Nutzer ergänzten Erweiterungen, soll immerhin die Topologie des Profils der SAF darstellen, die die Grundlage weiterer Ergänzungen baut.
